ethclient/ethclient.go

```diff
@@ -515,6 +515,21 @@ func (ec *Client) EstimateGas(ctx context.Context, msg ethereum.CallMsg) (uint64
 	return uint64(hex), nil
 }
 
+// CreateAccessList tries to create an access list for a specific transaction based on the
+// current pending state of the blockchain.
+func (ec *Client) CreateAccessList(ctx context.Context, msg ethereum.CallMsg) (*types.AccessList, uint64, string, error) {
```

Comment:
ethclient is designed for all eth-compatible clients. Is it true that `eth_createAccessList ` RPC endpoint is supported by all the (major) clients?